app/about-us/mobile.tsx

```diff
@@ -1,33 +1,57 @@
+'use client';
+
 import AboutImage from '@/images/about-image.png';
-import LeftSplash from '@/images/about-us-left-splash.png';
 import VideoBG from '@/images/about-us-video-bg.png';
 import VideoSplash from '@/images/about-us-video-splash.png';
+import { motion } from 'framer-motion';
 import Image from 'next/image';
 
+const textContent = [
+  {
+    title: 'TED',
+    desc: `TED merupakan sebuah organisasi nirlaba dengan semangat “Ideas Worth Spreading” yang berfokus pada Technology, Entertainment, dan Design. TED pertama kali dibentuk pada tahun 1984 dan menjadi ruang bagi individu-individu dengan semangat yang sama untuk menyebarkan ide-ide brilian di seluruh dunia.`,
+  },
+  {
+    title: 'TEDx',
+    desc: `TEDx merupakan events yang mengolaborasikan komunitas lokal untuk berbagi ide dengan konsep yang ada pada TED sehingga memicu diskusi dan koneksi yang mendalam. TEDx dengan label ikonik bersimbol “x” bermakna bahwa events tersebut diselenggarakan secara independen dengan lisensi dari TED.`,
+  },
+  {
+    title: 'TEDxUniversitasBrawijaya',
+    desc: `TEDxUniversitasBrawijaya merupakan salah satu dari TEDx event yang diselenggarakan secara independen dengan lisensi TEDx yang diinisiasi komunitas lingkup Universitas Brawijaya.`,
+  },
+];
+
+const textVariant = {
+  visible: { filter: `blur(0rem)` },
+  hidden: { filter: `blur(0.2rem)` },
+};
+
 function AboutUsMobile() {
   return (
     <main>
-      <section className="relative -z-30 flex h-fit w-full flex-col justify-center bg-wall bg-cover text-white">
+      <section className="relative -z-30 flex h-screen w-full flex-col justify-center bg-wall bg-cover text-white">
         <h1 className="mt-[4.75rem] px-5 text-center text-5xl sm:text-6xl">
           Tentang Kami
         </h1>
-        <div className="relative -bottom-10 aspect-[16/9] w-full">
+        <div className="relative -bottom-20 aspect-[2/3] w-full">
```

Comment:
![Screen Shot 2023-08-17 at 21 17 27](https://github.com/galihpermana29/tedx-frontend/assets/54941568/35fd3303-30af-4af2-af52-c3f191868123)

check the images, there is still a bug between the section 1 and the paper background.. see the attached images